hw/misc/hermes.c

```diff
@@ -174,6 +177,10 @@ struct hermes_dma_desc {
     hwaddr nxt_addr;
 };
 
+static inline size_t hw2ch(hwaddr addr){
+    return (addr & 0x1000) >> 8;
```

Comment:
That mask doesn't seems right to me, I think it should be `0xF00` (as the channel ID is bits 8:11)